translations/studio_api_docs.en.yaml

```diff
@@ -654,4 +654,10 @@ unit_quantity_value_convert_all_success_response: Converted quantity value
 data_object_get_select_options_description: |
   Get all dynamic select options for data objects for given field
 data_object_get_select_options_summary: Get all dynamic select options for data objects
-data_object_get_select_options_success_response: List of dynamic select options
\ No newline at end of file
+data_object_get_select_options_success_response: List of dynamic select options
+custom_reports_config_get_tree_description: Get all reports for the current user to display in configuration tree.
```

Comment:
I tried to sort the translation keys by alphabet

Reply:
It wasnt order by alphabet before (e.g. see asset_* keys on line 587 and version_* keys on 575)